universal-login-wallet/cypress/support/loginAppCommands.ts

```diff
@@ -1,30 +1,38 @@
 Cypress.Commands.add('approveTerms', () => {
+  cy.screenshot();
   cy.get('.terms-btn.button-primary').should('be.disabled');
   cy.get('#terms-label').find('[type="checkbox"]').check({ force: true }).should('be.checked');
   cy.get('#privacy-label').find('[type="checkbox"]').check({ force: true }).should('be.checked');
   cy.get('.terms-btn.button-primary').click();
 });
 
 Cypress.Commands.add('pickUsername', (name) => {
+  cy.screenshot();
```

Comment:
Where are we storing these screenshots? Do we actually need this functionality? Each time we run `cypress:open` we can see the whole clicking through the app and when we run ` cypress:run` we have. record of the process.

Reply:
We are storing screenshots in the /cypress/screenshots directory. Ok, maybe with recordings it might be not much useful.